flake.nix

```diff
@@ -82,6 +82,7 @@
               --ignore-engines --ignore-scripts
             patchShebangs .
 
+            yarn prebuild
```

Comment:
i thought `prebuild` scripts run automatically before `yarn build` without having to specify separately?

https://classic.yarnpkg.com/lang/en/docs/cli/run/#toc-yarn-run-script
<img width="804" alt="Screenshot 2024-11-08 at 7 12 52 AM" src="https://github.com/user-attachments/assets/bb47865b-2703-4552-a318-ca28a9c4fd47">

Reply:
Thanks for the review.
In the new [commit](https://github.com/fedimint/ui/pull/574/commits/698bc10d696cf66185986e872b93dd29ba3a6639), reorganized the version file and updated the imports.
- Moved `Version.tsx` from `./apps/router/src/home/` to `./apps/router/src/constants/` for better organization.
- Updated the import paths in the components to reflect the new location of `.version.tsx`.
- Ensured that the `prebuild` script generates the commit hash in the new location, maintaining availability during the build process.